chemclipse/tests/org.eclipse.chemclipse.msd.converter.fragment.test/src/org/eclipse/chemclipse/msd/converter/core/AbstractConverterSupport_6_Test.java

```diff
@@ -25,80 +30,81 @@ public class AbstractConverterSupport_6_Test extends AbstractConverterTestCase {
 	private ISupplier actSupplier;
 
 	@Override
-	protected void setUp() throws Exception {
+	@Before
+	public void setUp() throws Exception {
 
 		super.setUp();
 		converterSupport = getConverterSupport();
 		supplier = converterSupport.getSupplier();
 	}
 
-	@Override
-	protected void tearDown() throws Exception {
-
-		super.tearDown();
-	}
-
+	@Test
 	public void testGetSupplier_1() {
 
 		assertEquals(5, supplier.size());
 	}
 
+	@Test
 	public void testGetSupplier_2() {
 
 		String id = "org.eclipse.chemclipse.msd.converter.supplier.agilent";
 		String filterName = "Agilent Chromatogram (*.D/DATA.MS)";
 		try {
 			actSupplier = converterSupport.getSupplier(id);
 		} catch(NoConverterAvailableException e) {
-			assertTrue("The failure NoConverterAvailableException should not be thrown here.", false);
+			fail("NoConverterAvailableException");
```

Comment:
It is recommended to not catch and fail but rather change the test method to throw the exception. That way in case of failure one would get the stacktrace too.

Reply:
It should become just:
```
public void testGetSupplier_2() throws NoConverterAvailableException {

		String id = "org.eclipse.chemclipse.msd.converter.supplier.agilent";
		String filterName = "Agilent Chromatogram (*.D/DATA.MS)";
		actSupplier = converterSupport.getSupplier(id);
		assertEquals(filterName, actSupplier.getFilterName());
	}
```